readme.md

```diff
@@ -109,12 +109,14 @@ You can try `:help key` to know more about `key`. If it is an existing binding,
 
 #### Find mode
 
-Find mode is still incomplete and uses the Firefox feature "Quick Find". This will be improved eventually.
+Find mode is still incomplete and uses the Firefox feature "Quick find". This will be improved eventually.
```

Comment:
```suggestion
Find mode is still incomplete and uses the Firefox feature "Quick Find". This will be improved eventually.
```

"Quick Find" is what Mozilla calls it so we should stick to that : )

Reply:
Ah, I see that the Quick Find bar itself says `Quick find` in the search box, but I think that's just a normal imperative sentence, i.e. it has a capital "Q" just because it's the start of the sentence. So I still think we should call the bar the Quick Find bar :)